lib/utils.ts

```diff
@@ -80,7 +80,7 @@ export function maskRootdir(filepath: string): string {
                 .replace(/^\/app\//, '');
         } else {
             const re = getRegexForTempdir();
-            return filepath.replace(re, '/app/').replace(/^\/app\//, '');
+            return filepath.replace(/\\/, '/').replace(re, '/app/').replace(/^\/app\//, '');
```

Comment:
When using an objdump that is an exe it would return a path with `\`. This fixes that.

Reply:
does this work with paths on linux that have spaces? (cmdline the are sometimes written as `\ ` and it's a bit of strange escape exception. I guess it would be ok until users actually passed `\ ` and it might be ok to error on that)

Reply:
I didn't check with spaces. The wine clang objdump would send a path like `/something/something\main.cpp`